magento/adobe-stock-integration
# 1050: add delet button and functionality to the image details pop up

### Description (*)
This PR provides adding the Delete Image button with the related functionality to the view details image pop up.

### Fixed Issues (if relevant)
#1050 

### Manual testing scenarios (*)
1. Navigate to the stores settings and enable the enhanced media gallery: Stores -> Configuration -> Advanced -> System -> Enhanced Media Gallery -> Yes
2. Open any content where you can access the media files
3. Select image at the media grid
4. Press the View details link located under the ... button at every image
5. In the image details pop up press Delete Image button and check that image successfully deleted, pop up closed and a notification message is shown.
6. When you will test negative scenario for deleting image from the view details pop up, the pop up should stay open and the delete error notification message is shown in it.
